actions.go

```diff
@@ -88,15 +93,11 @@ func (c *Action) IssueCommand(cmd *Command) {
 // with the 'Command' argument as it's scope is unclear in the current
 // TypeScript implementation.
 func (c *Action) IssueFileCommand(cmd *Command) error {
-	return c.issueFileCommand(cmd, os.Getenv)
```

Comment:
I did a bunch of editorializing (i.e. removing the internal implementations that take a `getenvFunc`) but the changes can be removed if you want. I isolated the changes in small commits.

Reply:
This is fine.